features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
+Items can be bound to bindings i.e. for reading the status from e.g. Sonos, or for updating them. Read the wiki page for the respective binding for more help and examples.
+Items must also be defined in one or more files in the folder `${openhab_home}/conf/items`. Files here must have the extension .items but you can make as many .items files as you need/want however each item must be unique across them all.
```

Comment:
The path `${openhab_home}/conf/items` is not necessarily valid. On linux, it's `/etc/openhab2/items` for example. Maybe you better reduce this to "the `items` folder in your openhab configuration"?